dashboard/src/components/OperatorList/OperatorList.tsx

```diff
@@ -52,16 +52,27 @@ class OperatorList extends React.Component<IOperatorListProps> {
               </a>
             </div>
           </MessageAlert>
-          <LoadingWrapper loaded={!isFetching}>
-            {isOLMInstalled ? this.renderOperators() : <OLMNotFound />}
-          </LoadingWrapper>
+          <LoadingWrapper loaded={!isFetching}>{this.renderOperators()}</LoadingWrapper>
         </main>
       </div>
     );
   }
 
   private renderOperators() {
-    const { operators, error, csvs } = this.props;
+    const { operators, error, csvs, isOLMInstalled } = this.props;
+    if (error && error.constructor === ForbiddenError) {
+      return (
+        <ErrorSelector
+          error={error}
+          action="list"
+          resource="Operators"
+          namespace={this.props.namespace}
+        />
+      );
+    }
+    if (!isOLMInstalled) {
+      return <OLMNotFound />;
+    }
```

Comment:
Ah - is it because here we're only handling the one specific type of error, and ignoring any other, hence needing the OLMNotInstalled? Why not just handle the error?